src/pages/SignUp/SignUp.scss

```
@@ -0,0 +1,142 @@
+@import "../../styles/mixins.scss";
+
+.signUpForm {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  .signUpTitle {
+    margin-top: 30px;
+    padding: 10px 0;
+    text-align: center;
+    font-size: 40px;
+    font-weight: 700;
+  }
+
+  .accountContent {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 35%;
+    padding: 20px 0;
+    border-top: 1px solid rgba(0, 0, 0, 0.1);
+
+    .accountTitleProcess {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      width: 100%;
+      margin-bottom: 20px;
+
+      h3 {
+        font-size: 20px;
+        font-weight: 700;
+      }
+
+      .processBox {
+        display: flex;
+        justify-content: center;
+
+        .processUnderBar {
+          opacity: 0.1;
+        }
+
+        .processFirst,
+        .processSecond {
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          background-color: #ddd;
+          color: white;
+          width: 26px;
+          height: 26px;
+          border-radius: 50%;
+          font-size: 12px;
+          font-weight: 800;
+        }
+
+        .processSecond {
+          background-color: #2ac1bc;
+        }
+      }
+    }
+
+    .signUpInfoInputs {
+      width: 100%;
+
+      input {
+        width: 100%;
+        margin-bottom: 15px;
+        padding: 15px 8px;
+        border: 1px solid #ddd;
+        border-radius: 6px;
+
+        &:focus {
+          outline: none;
+        }
+      }
+
+      .inputIdBox,
+      .inputPwBox,
+      .inputPwCheckBox,
+      .inputAddressBox,
+      .inputEmailBox,
+      .inputNameBox,
+      .inputPhoneBox {
+        position: relative;
+
+        &::before {
+          @include inputInSideText();
+        }
+      }
```

```suggestion
     .같은속성을적용할className {
        position: relative;

        &::before {
          @include inputInSideText();
        }
      }
```
* `필수` 라는 메시지 출력이 반복적으로 필요하네요. className이 같다면 반복되는 속성은 동일하게 적용 됩니다.
* 그래서 공통으로 쓸 수 있는 className을 추가로 부여해서 스타일 적용하면 한줄로 해결 할 수 있겠습니다!

---

반영 완료 했습니다😀